week1/week1-homework.py

```diff
@@ -0,0 +1,72 @@
+# Home work for week1
+
+# 1. Print name and age
+name = 'Gebremedhin'
+age = 34
+print(f'My Name: {name}')
+print(f'My Age: {age}')
+
+# 2.  list of your 5 favorite movies
+favorite_movies = ['Spider man', 'Iron man', 'Salt', 'Titanic', 'Prison break']
+
+# 3. Displaying the first and last colors from the following list
+color_list = ["Red", "Green", "White", "Black"]
+first_and_last_color = []
+selected_color_indexes = [0, (len(color_list) - 1)]
```

Comment:
Good job! You can also access the last index of an array in python with `my_array[-1]` :)